### PR TITLE
Avatar & Badge Components

### PR DESCRIPTION
**Pull request by MarkVed17**
~ Added the Avatar Component
~ Added the Badge Component
~ Added main.css

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Looks good to go ! Amazing stylings.

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.avatar-indicator {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.rounded-circle {
+  position: absolute;
+  top: 95px;
+  left: 95px;
```
Comment (msdeshmukh009) [inline components/avatar/avatar.css]: Use rem if you are using it everwhere. That will make thing more consistent.

**Review by msdeshmukh009 — APPROVED** [review]
Looks good to go!!

**Review by srishti-maurya — APPROVED** [review]
UI looks good

**Comment by preethasri** [comment]
Looks good 👍

**Pull request merged**